sound/soc/sof/topology.c

```diff
@@ -2791,64 +2791,84 @@ static int sof_set_dai_config(struct snd_sof_dev *sdev, u32 size,
 	return 0;
 }
 
+static int sof_set_dai_config(struct snd_sof_dev *sdev, u32 size,
+			      struct snd_soc_dai_link *link,
+			      struct sof_ipc_dai_config *config)
```

Comment:
where is this called? it seems no one calls this function?

Reply:
@plbossart it is called in all other link_load functions than SSP (as this PR adds the multi config support for SSP only). So in essence this PR doesn't touch any other endpoints than SSP, even if you try to do multiple configs in other backend topologies they will only load 1 (first) config.